nilearn/plotting/tests/test_displays.py

```diff
@@ -38,3 +41,26 @@ def test_demo_ortho_projector():
     with tempfile.TemporaryFile() as fp:
         oprojector.savefig(fp)
     oprojector.close()
+
+
+# Smoke test to check threshold function for plot connectome
+def test_threshold_plot_connectome():
```

Comment:
Call it test_check_threshold. This doesn't have anything to do with plot_connectome any more.